src/main/java/unicash/model/category/UniqueCategoryList.java

```diff
@@ -199,4 +199,20 @@ public static boolean categoriesAreUnique(List<Category> categories) {
         }
         return true;
     }
+
+    /**
+     * Returns all categories as a single, unified string.
+     *
+     * @return all categories as a string
+     */
+    public String joinCategoriesAsString() {
+        String originalToString = this.toString();
+        String trimmedToString = originalToString
```

Comment:
Could we use the `internalList` directly instead? Using `toString()` makes it dependent on how `toString()` works so if we do change the implementation of `toString()`, it could break the joining

Reply:
Good idea, done.